imgui-render-opengl2/manifest

```diff
@@ -0,0 +1,12 @@
+: 1
+name: imgui-render-opengl2
+version: 1.86.0-a.0.z
+summary: imgui C++ library
```

Comment:
Same remarks as `imgui-core/manifest` , but also this needs a package-specific summary.